eden/scm/edenscm/git.py

```diff
@@ -128,6 +136,10 @@ def clone(ui, url, destpath=None, update=True, pullnames=None):
         repo = None
         shutil.rmtree(destpath, ignore_errors=True)
         raise
+    
+    if exists: 
+        shutil.move(temppath +"/.sl",destpath)
+        shutil.rmtree(temppath, ignore_errors=True)
```

Comment:
Should `temppath` by empty if `shutil.move()` succeeded?

Reply:
yes, the directory is empty after ``shutil.move()`` and therefore ``os.rmdir()`` could be used but since ``createrepo()`` works on empty folders this is no longer necessary